docs/mkdocs/docs/api/macros/json_disable_enum_serialization.md

```diff
@@ -0,0 +1,135 @@
+# JSON_DISABLE_ENUM_SERIALIZATION
+
+```cpp
+#define JSON_DISABLE_ENUM_SERIALIZATION
+```
+
+When defined, default parse and serialize functions for enums are excluded and have to be provided by the user, for example, using [`NLOHMANN_JSON_SERIALIZE_ENUM`](nlohmann_json_serialize_enum.md) (see [arbitrary type conversions](../../features/arbitrary_types.md) for more details).
+
+If you would try to parse or serialize enum you would get a compiler error, forcing you to implement enum serialization.
```

Comment:
```suggestion
Parsing or serializing an enum will result in a compiler error.
```

The rest of the sentence seems redundant given the sentence above.

Reply:
Changed